app/src/androidTest/java/com/github/sdpcoachme/schedule/ScheduleActivityTest.kt

```diff
@@ -263,7 +270,7 @@ class ScheduleActivityTest {
         val previousWeekEvent = EventOps.getPreviousWeekEvent()
 
         store.addEvent(previousWeekEvent, currentMonday).thenRun {
-            store.setCurrentEmail(defaultEmail)
+            store.setCurrentEmail(coachEmail)
```

Comment:
Be careful, setCurrentEmail returns a future. Would be better to apply a get(1, SECONDS) or use .thenApply before executing the rest of the code